public/javascripts/studentControllers.js

```diff
@@ -1,6 +1,11 @@
-var studentController = function($scope,  $http, $state) {
+$(document).ready(function(){
```

Comment:
this won't work anymore because we're removing `jQuery` from the project since you shouldn't use it along with angular